uqcs/templates/form.mako

```diff
@@ -66,34 +78,35 @@
             <label class="btn btn-primary">
```

Comment:
I think each of the button options should span the available horizontal space (not just the pay buttons). I know you seem to disagree with this strategy (or centralisation), so I can do this on my own time.

Reply:
Button groups are now full-width ([screenshot](https://user-images.githubusercontent.com/39479354/73931769-95da8d80-4924-11ea-8a36-86ed0fe1a7b1.png)). It does look neater.

Reply:
Much thanks. ❤️